ui/redux/reducers/collections.js

```diff
@@ -189,12 +200,13 @@ const collectionsReducer = handleActions(
     },
     [ACTIONS.USER_STATE_POPULATE]: (state, action) => {
       const { builtinCollections, savedCollections, unpublishedCollections, editedCollections } = action.data;
+
       return {
         ...state,
         edited: editedCollections || state.edited,
         unpublished: unpublishedCollections || state.unpublished,
         builtin: builtinCollections || state.builtin,
-        saved: savedCollections || state.saved,
+        saved: Object.assign(state.saved, savedCollections),
       };
     },
```

Comment:
:small_red_triangle: Changes must be immutable

The use of `Object.assign` on an array ended up altering the first argument `state.saved` directly.

<img width="236" alt="image" src="https://user-images.githubusercontent.com/64950861/180915050-b8fe6eb9-4ea9-4bca-8074-9870f603f416.png">

Feels like the original code is correct (use `savedCollections`, or fallback to original `state.saved` if no changes), unless I missed something.